ci/ci.py

```diff
@@ -343,18 +353,18 @@ def main(argv):
         make_release = True
         engine_artifacts = args.engine_artifacts or "archived"
     elif branch == "editor-dev":
-        engine_channel = "stable"
+        engine_channel = "editor-alpha"
         editor_channel = "editor-alpha"
         release_channel = editor_channel
         make_release = True
         engine_artifacts = args.engine_artifacts
     elif branch and branch.startswith("DEFEDIT-"):
-        engine_channel = None
+        engine_channel = "editor-dev"
         editor_channel = None
         make_release = False
         engine_artifacts = args.engine_artifacts or "archived-stable"
     else: # engine dev branch
-        engine_channel = None
+        engine_channel = "engine-dev"
```

Comment:
If this is "engine-dev", shouldn't the others me engine-beta and engine-master?

Reply:
Hmm, I'm thinking that the beta isn't really specific to engine or editor. It's the merged result of engine and editor development.

Reply:
Same could be said with "dev" (or the previous name "alpha": http://d.defold.com/alpha), it's the merged result of the editor-dev and dev. The merge is done at the start of every beta.

One could also imagine the editor being released under "editor".
We don't really use any other editor channel.

Here I don't have a strong opinion though.